tests/functional/object/mcg/ui/test_s3_vector_ui.py

```
@@ -431,3 +431,149 @@ def teardown():
+                vectorBucketName=noobaa_bucket_name,
+            )
+            logger.info(f"Pre-req: created index '{index_name}'")
+
+        # Step 5: Navigate to the bucket detail page.
+        # The S3 Vector tab displays the NooBaa internal bucket name.
+        vector_ui = buckets_tab.nav_s3_vector_tab()
+        vector_ui.navigate_to_vector_bucket(noobaa_bucket_name)
+
+        # Steps 6-7: Delete the first index via the row kebab menu.
+        index_to_delete = created_index_names[0]
+        logger.info(f"Steps 6-7: Deleting index '{index_to_delete}' via UI")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)